src/sql.rs

```diff
@@ -721,13 +733,6 @@ pub async fn housekeeping(context: &Context) -> Result<()> {
         warn!(context, "Failed to run incremental vacuum: {}", err);
```

Comment:
unrelated to the pr, but i am always getting the warning `Failed to run incremental vacuum: Execute returned results - did you mean to call query?` here.

Reply:
yeah, thought so :)

thanks for the fix!